site/app/views/submission/TeamView.php

```diff
@@ -21,6 +21,7 @@ public function showTeamPage(Gradeable $gradeable, $team, $teams, $lock, $users_
         $seekers = [];
         $invites_received = [];
         $seeking_partner = false;
+        $this->core->getOutput()->addBreadcrumb("Manage Team for {$gradeable->getTitle()}");
```

Comment:
This looks good the only thing I would say is that the text already on the page says something like `Manage Team For: Open Team Homework` however the breadcrumb looks like `Manage Team for Open Team Homework` I think it makes more sense to change the `for` in your breadcrumb to `For:`

![image](https://user-images.githubusercontent.com/18558130/70738849-91f30c00-1ce3-11ea-87ac-7b814b485302.png)

Reply:
I changed it to `Manage Team For: `
![image](https://user-images.githubusercontent.com/25368899/70740086-63c2fb80-1ce6-11ea-99d6-3fba3e2ebed3.png)